kolibri/core/templates/kolibri/base.html

```diff
@@ -1,10 +1,8 @@
-{% load i18n kolibri_tags %}
-<html>
+{% load i18n kolibri_tags webpack_tags %}<html>
 <head>
   <title>{% block title %}{% endblock %} - {% trans "Kolibri" %}</title>
-  {% frontend_assets 'kolibri.core.KolibriCoreFrontEnd' %}
-  {% base_frontend_async %}
-  {% base_frontend_sync %}
+  {% webpack_assets 'default_frontend' %}
+  {% base_frontend_assets %}
```

Comment:
> But maybe there are better naming conventions?

The name seems fine to me. But is it functionally the same? Or did you remove the distinction between sync and async assets somehow?